orchestra/analytics/latency.py

```diff
@@ -51,6 +53,8 @@ def work_time_df(projects, human_only=True, complete_tasks_only=True):
             human_only, complete_tasks_only)
         for project in projects))
     df = DataFrame(time_data)
+    df['start_datetime'] = df['start_datetime'].astype('datetime64[ns]')
```

Comment:
What does this do? Why were things working without it? Maybe an explanatory comment?